### PR TITLE
Edevate 396

### PR DESCRIPTION
**Pull request by inlanger**

### CODE REVIEW

File: common/djangoapps/student/views.py [hunk 1]
```diff
@@ -1093,6 +1104,42 @@ def change_enrollment(request, check_access=True):
...
+    # Get the user
+    user = request.user
+
+    # Ensure the user is authenticated
+    if not user.is_authenticated():
+        return HttpResponseForbidden()
+
+    order_date = request.POST.get("order_date")
+    order_status = request.POST.get("order_status")
+    if order_status == 'Complete':
```
Comment (aniav) [inline common/djangoapps/student/views.py:1130]: You might want to import `COMPLETE` from `lms/commerce/constants.py` if this is the same constant :)

**Comment by aniav** [comment]
A lot of code I don't know, but LGTM :)

File: common/djangoapps/student/models.py [hunk 2]
```diff
@@ -398,6 +398,21 @@ def country_cache_key_name(cls, user_id):
         return cls.PROFILE_COUNTRY_CACHE_KEY.format(user_id=user_id)
 
 
+class Subscriber(models.Model):
+    user = models.OneToOneField(User, unique=True, db_index=True,
```
Comment (CrowbarKZ) [inline common/djangoapps/student/models.py:402]: it's not necessary to specify `db_index=True` when `unique=True` is present

File: lms/djangoapps/verify_student/views.py [hunk 3]
```diff
@@ -699,6 +699,176 @@ def _response_if_deadline_passed(self, course, deadline_name, deadline_datetime)
...
+            request (HttpRequest): The request object.
+
+        Returns:
+            HttpResponse
+
+        Raises:
+            Http404: The course does not exist.
+        """
+        # Get the course key
+        course_key = CourseKey.from_string(settings.SUBSCRIPTION_COURSE_KEY)
```
Comment (CrowbarKZ) [inline lms/djangoapps/verify_student/views.py:719]: Just trying to understand :) what is the `SUBSCRIPTION_COURSE_KEY` ? some special course we create to test if users have access to it?

**Comment by CrowbarKZ** [comment]
LGTM :)

**Pull request merged**